### PR TITLE
Remove CI table from `functions_framework` readme

### PR DESCRIPTION
**Pull request by kevmoo**
It's fine just to have this in the root, right?

### CODE REVIEW

**Comment by kevmoo** [comment]
Feel free to squash/merge and delete the branch if you're happy @subfuzion

**Comment by subfuzion** [comment]
Personally, I like having status in this repo without having to refer to the parent repo. I believe most of the other frameworks (at least Go, Node, and Python) display some level of build success plus package badges, etc, directly from their repos as well. Any particular reason you feel it should be removed?

**Review by subfuzion — COMMENTED** [review]
Personally, I like having status in this repo without having to refer to the parent repo. I believe most of the other frameworks (at least Go, Node, and Python) display some level of build success plus package badges, etc, directly from their repos as well. Any particular reason you feel it should be removed?

**Comment by subfuzion** [comment]
I do like seeing them here at a glance, but since in theory nothing gets merged to main until all the tests pass anyway, would you prefer to see something a bit more consolidated? And how about pub package badges? A few examples:

[pub package](https://pub.dev/packages/functions_framework)

[pub points](https://pub.dev/packages/functions_framework/score)

[popularity](https://pub.dev/packages/functions_framework/score)

[likes](https://pub.dev/packages/functions_framework/score)

**Comment by kevmoo** [comment]
@subfuzion – pub badge added

**Comment by subfuzion** [comment]
Rationale?

**Comment by kevmoo** [comment]
> Rationale?

Oh...just to make the readme more clean. It's not helpful for folks looking at packages IMHO. For the repo, sure.

**Comment by subfuzion** [comment]
I get the idea of making the readme cleaner. It does feel cluttered. I guess what I meant to say earlier is that I do like seeing repos that have some kind of "build passing" status (in addition to a package badge). Can we consolidate all the test badges with a single status badge just to inspire confidence to visitors here? Maybe along the lines of these other frameworks ([ex 1](https://github.com/GoogleCloudPlatform/spring-cloud-gcp), [ex 2](https://github.com/GoogleCloudPlatform/webapp2), and all the other individual function framework repos)?

**Review by grant — DISMISSED** [review]
Sounds good. We can keep the badges in the main FF table.

https://github.com/GoogleCloudPlatform/functions-framework

File: functions_framework/README.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 > DISCLAIMER: this is not ready for production.
 
-|Functions Framework|Unit Tests|Lint Test|Conformance Tests|
-|---|---|---|---|
-[Dart][ff_dart]| [![][ff_dart_unit_img]][ff_dart_unit_link] | [![][ff_dart_lint_img]][ff_dart_lint_link] | [![][ff_dart_conformance_img]][ff_dart_conformance_link] |
```
Comment (grant) [inline functions_framework/README.md:7]: Might want to delete the links too, like `ff_dart_conformance_img` in this file.

Comment (subfuzion) [reply functions_framework/README.md:7]: Don't want to hold the PR up, but I personally like the idea of at least one badge to indicate the build is good -- i always like seeing that in other repos (even for library packages, not just apps). Not every visitor to the repo is aware of the main functions framework repo or even of our CI process for pull requests.

**Comment by grant** [comment]
Can we just leave the CI badges and add the pub badge? They're all useful.

Screen Shot 2020-12-16 at 12 38 01

**Review by subfuzion — APPROVED** [review]
Personally, I would have preferred leaving a "build status" badge in here along with a package badge, but don't want to hold this up. If after consideration, you're both (@kevmoo and @grant) okay with this, then let's merge. Can always add a badge back later if we want!

**Review by grant — CHANGES_REQUESTED** [review]
Suggested keeping the table per comment above.

**Pull request closed without merging**